app/Repositories/JsonBaseRepository.php

```
@@ -44,7 +57,7 @@ public function allGroupedBy($field)
             throw new InvalidFieldException($field);
```

please throw this error inside the getGroupedBy method